WordPress/src/main/java/org/wordpress/android/ui/prefs/DeleteSiteDialogFragment.java

```diff
@@ -108,14 +108,13 @@ private Spannable confirmationPromptString() {
     private void configureUrlConfirmation(AlertDialog.Builder builder) {
         View view = getActivity().getLayoutInflater().inflate(R.layout.delete_site_dialog, null);
         mUrlConfirmation = (EditText) view.findViewById(R.id.url_confirmation);
-        mUrlConfirmation.setHint(mSiteDomain);
```

Comment:
Not a big deal, but I kind of liked showing the the domain as the hint. Why was this dropped?